zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/stream/JobClientStreamMetricsDoc.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.impl.stream;
+
+import io.camunda.zeebe.util.micrometer.ExtendedMeterDocumentation;
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.core.instrument.Meter.Type;
+
+@SuppressWarnings("NullableProblems")
+public enum JobClientStreamMetricsDoc implements ExtendedMeterDocumentation {
+  /** The count of known job stream servers/brokers */
+  SERVERS {
+    @Override
+    public String getDescription() {
+      return "The count of known job stream servers/brokers";
+    }
+
+    @Override
+    public String getName() {
+      return "zeebe.gateway.job.stream.servers";
+    }
+
+    @Override
+    public Type getType() {
+      return Type.GAUGE;
+    }
+  },
+
+  /** The count of known job stream clients */
+  CLIENTS {
+    @Override
+    public String getDescription() {
+      return "The count of known job stream clients";
+    }
+
+    @Override
+    public String getName() {
+      return "zeebe.gateway.job.stream.clients";
+    }
+
+    @Override
+    public Type getType() {
+      return Type.GAUGE;
+    }
+  },
+
+  /** Total count of aggregated streams */
+  AGGREGATED_STREAMS {
+    @Override
+    public String getDescription() {
+      return "Total count of aggregated streams";
+    }
+
+    @Override
+    public String getName() {
+      return "zeebe.gateway.job.stream.streams";
+    }
+
+    @Override
+    public Type getType() {
+      return Type.GAUGE;
+    }
+  },
+
+  /** Distribution of client count per aggregated stream */
+  AGGREGATED_CLIENTS {
```

Comment:
:wrench: I think we need to add the distributionSLOs as the default is empty? Not sure if it's needed though

Reply:
Can you take this over? Check what are the default right now with Prometheus, and just use them?